sdk/tests/deqp/framework/common/tcuFloat.js

```diff
@@ -434,7 +466,11 @@ tcuFloat.newDeFloatFromParameters = function(jsnumber, description) {
  * @return {number}
  */
 tcuFloat.deFloat.prototype.getBitRange = function(begin, end) {
-    return deMath.getBitRange(this.bits(), begin, end);
```

Comment:
`bits()` is invalid here if this is a float64. So this has to be done on an alternate representation, like the ArrayBuffer.